website_odoo_debranding/readme/USAGE.rst

```diff
@@ -1,3 +1 @@
-#. Go to your home page.
-#. Enable or disable *Customize > Footer Copyright > Remove Odoo Promotional
-   Link* at your will. It should be enabled by default.
+#.No additional configuration is needed. The legend "Powered by Odoo" from footer is removed automatically.
```

Comment:
Here's the missing space. Anyway, why this change of behavior?

Reply:
Now you can control the visibility in the "Customize" menu. Updated readme.